Resources/services.xml

```diff
@@ -39,5 +43,22 @@
         <service id="shyim_webp.commands.generate_webp" class="ShyimWebP\Commands\GenerateWebpImages">
             <tag name="console.command"/>
         </service>
+
+        <service id="shyim_webp.commands.download_google_binaries" class="ShyimWebP\Commands\DownloadGoogleBinaries">
+            <tag name="console.command"/>
+        </service>
+
+        <service id="shyim_webp.services.webp_encoder_factory" class="ShyimWebP\Services\WebpEncoderFactory">
+            <argument type="tagged" tag="shyim_webp.webp_encoder"/>
```

Comment:
That breaks compatibility to lower 5.5 versions

Reply:
It does. So rather implement it with a `Plugin::build` and the `TagReplaceTrait` or set the minimum shopware version higher?

Reply:
For "just" Tagged Services i don't want really drop support for older reasons if that is the only one reason.

Yep build method a new compilerpass with the TagReplaceTrait :)